### PR TITLE
docs: show version Number features were added

### LINKED ISSUE #17054: Update website APIs to show version Number features were added.
- [x] I have searched the [issues](https://github.com/ant-design/ant-design/issues) of this repository and believe that this is not a duplicate.

### What problem does this feature solve?
I'm using An older version of Ant Design (^3.7.3) 
 There have been so many updates to the components that I often get lost in the API Docs, and try using things as they develop.  
 It would be great to know when an API was updated so I know instantly if I have access to a feature.

While I know using the latest version of Ant Design would be the best thing to do, unfortunately it is not a feasible solution. 

### What does the proposed API look like?

# API
## Table
			

|Property   	| Description  	|  Type 	| Default  	|  Version Added  	|
|---	|---	|---	|---	|---	|
|  bordered 	|  Whether to show all table borders 	|   boolean	|  false         |  x.xx.x

### PR DESCRIPTION
**Pull request by muzea**
### 🤔 This is a ...

- [X] Site / document update

### 👻 What's the background?

fix https://github.com/ant-design/ant-design/issues/17054

数据来源于文档中第一次出现的版本tag，会因为文档结构变化导致版本号计算错误。

rebase了一次master，整个人都不好了。
目前遗留的问题

- ~~notification~~/form 的类型因为有prop重名所以可能是不对的
- 出现的3.19.8这个版本是因为有的修改没有发

- - - -

The data comes from the version tag that first appeared in the document, and the version number is incorrectly calculated due to changes in the document structure.

Rebase to the master is really too painful.
Current issues

- The type of ~~notification~~/form may be incorrect because some prop name is duplicated.
- The version 3.19.8 that appears is because Some modifications are not released now.

### CODE REVIEW

**Comment by muzea** [comment]
May be affected by #17365.

File: components/anchor/index.en-US.md [hunk 1]
```diff
@@ -15,19 +15,19 @@ For displaying anchor hyperlinks on page and jumping between them.
...
-| Property | Description | Type | Default |
-| --- | --- | --- | --- |
-| affix | Fixed mode of Anchor | boolean | true |
-| bounds | Bounding distance of anchor area | number | 5(px) |
-| getContainer | Scrolling container | () => HTMLElement | () => window |
-| offsetBottom | Pixels to offset from bottom when calculating position of scroll | number | - |
-| offsetTop | Pixels to offset from top when calculating position of scroll | number | 0 |
-| showInkInFixed | Whether show ink-balls in Fixed mode | boolean | false |
-| onClick | set the handler to handle `click` event | Function(e: Event, link: Object) | - |
+| Property | Description | Type | Default | Version Added |
```
Comment (afc163) [inline components/anchor/index.en-US.md]: Added 不是很准确，很多是 2.x 就支持了的。直接 version 就好了。

File: components/auto-complete/index.en-US.md [hunk 2]
```diff
@@ -18,33 +18,33 @@ const dataSource = ['12345', '23456', '34567'];
...
-| allowClear | Show clear button, effective in multiple mode only. | boolean | false |
-| autoFocus | get focus when component mounted | boolean | false |
-| backfill | backfill selected item the input when using keyboard | boolean | false |
-| children (for customize input element) | customize input element | HTMLInputElement / HTMLTextAreaElement / React.ReactElement<InputProps> | `<Input />` |
+| Property | Description | Type | Default | Version Added |
+| --- | --- | --- | --- | --- |
+| allowClear | Show clear button, effective in multiple mode only. | boolean | false | 3.0.0 |
+| autoFocus | get focus when component mounted | boolean | false | 3.0.0 |
+| backfill | backfill selected item the input when using keyboard | boolean | false | 3.0.0 |
+| children (for customize input element) | customize input element | HTMLInputElement / HTMLTextAreaElement / React.ReactElement<InputProps> | `<Input />` | 3.0.0 | 3.0.0 |
```
Comment (afc163) [inline components/auto-complete/index.en-US.md]: 多了一列

File: components/anchor/index.en-US.md (hunk 1 above)
Comment (muzea) [reply components/anchor/index.en-US.md]: 是考虑到文档更新也只会覆盖到3.x，回去算2.x的文档信息参考价值就没有那么大了。
就统一表头 Version Added 到 Version 嘛？

Comment (afc163) [reply components/anchor/index.en-US.md]: 👌

**Comment by muzea** [comment]
🌚 又要rebase master了

**Comment by muzea** [comment]
notification 核对了一下，没啥问题了。
form 东西比较多，粗略核对过是没啥问题的。

**Comment by muzea** [comment]
@afc163 这个PR现在还有什么问题吗？

**Comment by afc163** [comment]
有冲突了。

**Comment by muzea** [comment]
等我喝瓶快乐水再干活

**Comment by muzea** [comment]
@afc163 done

**Comment by muzea** [comment]
😢 怎么又冲突了

**Comment by afc163** [comment]
解决一下我今天合掉。

**Comment by muzea** [comment]
@afc163 冲突解决了

**Comment by dengfuping** [comment]
@muzea It seems that no need to add `3.0.0` version. It's enough to just add special versions.

**Comment by muzea** [comment]
@dengfuping There are two considerations here.
- Calculate which version of a prop is dependent on the structure of the document
- Which specific version of a prop is added from 2.x or 1.x, which is of little significance to the current user

**Comment by dengfuping** [comment]
@muzea IMO:
- Documents of every major version is deployed separately.
- Low readability for obvious version `3.0.0`.

**Comment by muzea** [comment]
@dengfuping "3.0.0" just wants to express that every version in 3.x, this prop is available. It means that if leave it blank, will better express this meaning?

**Comment by afc163** [comment]
Leaving it blank would be better, it make version information highlighted and easy to find.

**Comment by muzea** [comment]
@afc163 @dengfuping Removed meaningless “3.0.0”.

**Pull request merged**